bin/p2-replicate/main.go

```diff
@@ -77,6 +87,27 @@ func main() {
 	replicator.Logger.Logger.Formatter = new(logrus.TextFormatter)
 	replicator.MinimumNodes = *minNodes
 
+	if *skip {
+		status, err := healthChecker.LookupHealth(manifest.ID())
+		if err != nil {
+			replicator.Logger.WithField("err", err).Errorln("Could not retrieve health checks to resolve skipped nodes")
+			os.Exit(1)
+		}
+
+		var newHosts []string
+		for _, host := range *hosts {
+			thisHealth, _ := status.ForNode(host)
+			if thisHealth.Health != health.Passing && thisHealth.Health != health.Unknown {
+				replicator.Logger.WithFields(logrus.Fields{
+					"host":   host,
+					"health": thisHealth.Health,
+				}).Infoln("Skipping unhealthy host")
+			}
+			newHosts = append(newHosts, host)
+		}
+		replicator.Allocation = allocation.NewAllocation(newHosts...)
```

Comment:
I'm having a somewhat difficult time wrapping my head around the `skip` semantics. Is skip intended to ignore hosts that are as or less healthy than the `threshold`? I imagine you'd not want to skip the nodes if they are out of date anyway.